oslabs-beta/DockerLocal
# Fetch repos and add to projects

## Description:
- These changes add functionality to populate the AddRepos modal with data from fetch request and add repos to the selected project. It also implements our initial color scheme.
<img width="815" alt="Screen Shot 2020-07-01 at 3 28 22 PM" src="https://user-images.githubusercontent.com/64039601/86297309-cfe84c80-bbaf-11ea-810f-514e5554f84c.png">

## Changes I Made:
- Change color scheme (not final version)
- Filter data from fetch request and populate AddRepos modal
- AddRepos modal 'Add Repos' button changes state of currently selected project
- AddRepos will not let user add repos if they already exist in the current project
- Repo type changed to reflect the data we're getting from requests and using in shell scripts
## How to Test:
- npm install and npm start to launch the app
- Click sign in with Github (this is still buggy, sometimes you need to click more than once)
- Click toggle isLoggedIn dev button, then Add Repositories to open the AddRepos modal
- The modal should populate with data from github (if it doesn't, navigate back and sign in with Github again)
- Click the checkmark on the AddRepos modal, then 'Add Repos' and the selected repos should be on the project page.
